Demo/Common/IntegrationExamples/Protocols/DropInExampleProtocol.swift

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import AdyenNetworking
+import AdyenSession
+
+internal protocol DropInExampleProtocol: APIClientAware {
+    var context: AdyenContext { get }
+    func requestAdyenSessionConfiguration(completion: ((AdyenSession.Configuration?, Error?) -> Void)?)
+}
+
+extension DropInExampleProtocol {
+
+    internal var context: AdyenContext {
+        var analyticsConfiguration = AnalyticsConfiguration()
+        analyticsConfiguration.isEnabled = true
+        return AdyenContext(apiContext: ConfigurationConstants.apiContext,
+                            payment: ConfigurationConstants.current.payment,
+                            analyticsConfiguration: analyticsConfiguration)
+    }
+
+    internal func requestAdyenSessionConfiguration(completion: ((AdyenSession.Configuration?, Error?) -> Void)?) {
+        let request = SessionRequest()
+        apiClient.perform(request) { result in
```

Comment:
`[weak self]` since self is captured

Reply:
Ah! I missed it. Fixed this da093c2e272a314effba7b3c43882271964a0529

Reply:
This still does not have [weak self]

Reply:
This is not a class.[weak self] is only applied to classes to remove reference cycle.

Reply:
Ah didn't realize that. But the instance calling this will be a class and thus could fall into the capture. 🤔 

We could add `AnyObject` to this protocol and implement the capture semantics. In our case only classes would inherit this protocol anyway.

Reply:
Yes, you have a point. I have made this change 7635bc8fee151fe57becc96e6c6626975f37b3d3